pages/project/[project].tsx

```diff
@@ -16,10 +16,12 @@ export const getStaticProps = async (context) => {
   const worksListPath = path.join(process.cwd(), "public/info/works.json");
   const projects = await fs.readFile(worksListPath, "utf-8");
   const allProjectsDict = JSON.parse(projects);
-  const allProjects = Object.keys(allProjectsDict).map((url) => ({
-    ...allProjectsDict[url],
-    url,
-  }));
+  const allProjects = Object.keys(allProjectsDict)
+    .map((url) => ({
+      ...allProjectsDict[url],
+      url,
+    }))
+    .filter((project) => project.hide === false);
```

Comment:
```suggestion
    .filter((project) => !project?.hide);
```

Also hide should be an optional field in a project. If hidden, have it. Otherwise not necessary.